### DISCUSSION TITLE
Exclude specific submodules from the documentation

### DISCUSSION THREAD

**Discussion by ldouteau** [mkdocstrings/python#185] [Q&A] [answered]
Hi,
Is there a way to prevent specific submodules from being documented ? E.g., with the following structure, can i get `::: a` to document submodule `b` but not `c`

```
📁 a/
├── 📁 b/
└── 📁 c/
```

**Comment by ldouteau** [comment, marked as answer]
Well I managed to make this work with [`filters`](https://mkdocstrings.github.io/python/usage/configuration/members/#filters)

**Reply by pawamoy** [reply]
Ah, just saw this. Great!

**Comment by pawamoy** [comment]
Hi! You can try using `members` or `filters`:

```md
::: a
    options:
        members: [b]
```

```md
::: a
    options:
        filters: ["!^c$"]
```
